codegen-luau/runtime/numeric.lua

```diff
@@ -25,7 +25,7 @@ local math_pow = math.pow
 local table_freeze = table.freeze
 
 local from_u32, into_u32, from_u64, into_u64
-local num_add, num_subtract, num_multiply, num_divide_unsigned, num_negate
+local num_add, num_subtract, num_multiply, num_divide_unsigned, num_negate, num_bit_not
```

Comment:
Last name related thing, but it's probably preferable it be `num_not` as there is no other way to "not" a number.